omdv-build-iso.sh

```diff
@@ -1249,6 +1250,10 @@ setupGrub2() {
 	    errorCatch
 	fi
     fi
+# Fix up 2014.0 grub installer line...We don't have Calamares in 2014.
+    if [ $VERSION == OPENMANDRIVA2014.0 ]; then
```

Comment:
Whoops!  Forgot to hit the caps lock key.
Will fix,
Thanks,
Colin


On Sunday, 24 July 2016 03:57:46 BST Tomasz Paweł Gajc wrote:
> > @@ -1249,6 +1250,10 @@ setupGrub2() {
> >  	    errorCatch
> >  	fi
> >      fi
> > +# Fix up 2014.0 grub installer line...We don't have Calamares in 2014.
> > +    if [ $VERSION == OPENMANDRIVA2014.0 ]; then
> 
> @itchka  Please when do comparin do it on lowercase... keep consistency with rest of the code.
> 
> ${version,,} =~ openmandriva201.* 
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/OpenMandrivaAssociation/omdv-build-iso/pull/34/files/ab247063c3c56dcefd7a3c608f5f8d0ef9ab046b..3d99ca10a90aee65cb0854551023882df5e5c6db#r71986586